runtime/tests/jni/module.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <!--
-   Copyright (c) 2006, 2018 IBM Corp. and others
+   Copyright (c) 2006, 2019 IBM Corp. and others
```

Comment:
Pls remove this unnecessary change.

Reply:
Without this change, the copyright check fails. Is there a better way to prevent that from happening?

Reply:
It won't fail if you remove this change, because you've removed the other change in this file as well.